nifi-commons/nifi-properties/src/main/java/org/apache/nifi/util/NiFiProperties.java

```diff
@@ -1540,6 +1540,12 @@ public boolean isZooKeeperTlsConfigurationPresent() {
             && getProperty(NiFiProperties.ZOOKEEPER_SECURITY_TRUSTSTORE_PASSWD) != null;
     }
 
+    public boolean isTlsConfigurationPresent() {
+        return StringUtils.isNotBlank(getProperty(NiFiProperties.SECURITY_KEYSTORE))
+            && getProperty(NiFiProperties.SECURITY_KEYSTORE_PASSWD) != null
```

Comment:
Is there a reason for just checking whether the keystore password is not null, as opposed to checking whether it is not blank?

Reply:
Only because I think technically the keystore/truststore passwords can be empty string.